Automattic/sensei
# Remove `[sensei_unpurchased_courses]` shortcode

This PR:
- Removes `[sensei_unpurchased_courses]` shortcode.
- Adds new filter so add-ons can use Sensei's shortcode handler.
- Moves shortcode initialization further down the bootstrap flow into `init`.

### Testing Instructions
- Make sure other shortcodes (`[sensei_featured_courses]`, `[sensei_user_courses]`, etc) are unaffected by the change in boot order.

### New Filters
- `includes/shortcodes/class-sensei-shortcode-loader.php`: `sensei_shortcode_classes`
